actionpack/lib/action_dispatch/middleware/host_authorization.rb

```diff
@@ -4,7 +4,14 @@
 
 module ActionDispatch
   # This middleware guards from DNS rebinding attacks by explicitly permitting
-  # the hosts a request can be sent to.
+  # the hosts a request can be sent to. Options are passed to this middleware
+  # from +config.host_authorization+ and can be used to configure the response
+  # application and an exclude callable.
+  #
+  # Requests can opt-out of Host Authorization by passing a callable object in
+  # the exclude setting:
```

Comment:
```suggestion
  # Requests can opt-out of Host Authorization with +exclude+:
```

I don't think we need to mention "callable" here; the example below is enough.